android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerSimpleCache.kt

```diff
@@ -0,0 +1,31 @@
+package com.brentvatne.exoplayer
+
+import android.content.Context
+import androidx.media3.database.StandaloneDatabaseProvider
+import androidx.media3.datasource.DataSource
+import androidx.media3.datasource.HttpDataSource
+import androidx.media3.datasource.cache.CacheDataSource
+import androidx.media3.datasource.cache.LeastRecentlyUsedCacheEvictor
+import androidx.media3.datasource.cache.SimpleCache
+import java.io.File
+
+object RNVSimpleCache {
+    // TODO: when to release? how to check if cache is released?
+    var simpleCache: SimpleCache? = null
+    var cacheDataSourceFactory: DataSource.Factory? = null
+
+    fun setSimpleCache(context: Context, cacheSize: Int, factory: HttpDataSource.Factory) {
+        if (cacheDataSourceFactory != null || cacheSize == 0) return
```

Comment:
I am not sure of this condition ... can you double check please ?
* if cacheSize == 0 -> I think you can clear simpleCache and cacheDataSourceFactory (just set them to 0). Can this condition block deactivation of caching ?
* cacheDataSourceFactory != null -> this condition will disallow caching cache size ?

Reply:
@lovegaoshi reminder on this point !
But I thought a little bit more:
- cache shouldn't be removed (ie: I don't think cache size = 0 should clean it)
- Maybe the cache size should be an external property (out of buffer props) ... 
On iOS cache size is hard coded:
options.sizeConstraintBytes = 1024 * 1024 * 100;
Let me know your point of view please !